gallery/src/styles/NxFormLayout/NxFormLayoutPage.tsx

```diff
@@ -102,6 +87,23 @@ const NxFormLayoutPage = () =>
               radio buttons.
             </td>
           </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-legend</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Standard class for <code className="nx-code">&lt;legend&gt;</code> elements. A legend is used inside of a
+              <code className="nx-code">&lt;fieldset&gt;</code> in the place of a
+              <code className="nx-code">&lt;label&gt;</code>
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-legend--optional</code></td>
+            <td className="nx-cell">Modifier</td>
+            <td className="nx-cell">
+              Used when you want "- optional" text to appear after a <code className="nx-code">&lt;legend&gt;</code>
```

Comment:
Here and on `.nx-label--optional`, I think the quoted text is out of date.
```suggestion
              Used when you want "Optional" text to appear after a <code className="nx-code">&lt;legend&gt;</code>
```

Reply:
and: https://github.com/sonatype/sonatype-react-shared-components/pull/135/commits/257fb491da8355c2c7ebd2c0d929be5b107e2d67